haystack/nodes/connector/crawler.py

```diff
@@ -101,21 +101,21 @@ def __init__(
 
         IN_COLAB = "google.colab" in sys.modules
         IN_AZUREML = os.environ.get("AZUREML_ENVIRONMENT_IMAGE", None) == "True"
-        IS_ROOT = sys.platform not in ["win32", "cygwin"] and os.geteuid() == 0  # type: ignore   # This is a mypy issue of sorts, that fails on Windows.
+        IN_WINDOWS = sys.platform in ["win32", "cygwin"]
+        IS_ROOT = not IN_WINDOWS and os.geteuid() == 0  # type: ignore   # This is a mypy issue of sorts, that fails on Windows.
 
         if webdriver_options is None:
             webdriver_options = ["--headless", "--disable-gpu", "--disable-dev-shm-usage", "--single-process"]
-        elif "--headless" not in webdriver_options:
-            webdriver_options.append("--headless")
-
-        if IS_ROOT and "--no-sandbox" not in webdriver_options:
+        webdriver_options.append("--headless")
+        if IS_ROOT or IN_WINDOWS:
             webdriver_options.append("--no-sandbox")
-
-        if (IN_COLAB or IN_AZUREML) and "--disable-dev-shm-usage" not in webdriver_options:
+        if IN_WINDOWS:
+            webdriver_options.append("--no--remote-debugging-port=9222")
```

Comment:
Could you add to the docs strings why this is a necessary option for running the Crawler in Windows?

Reply:
There was an error on the option. I fixed it.
Furthermore, rereading the Stack Overflow thread, I think this option can be safely adopted both in Windows and Linux (root), so I modified my code.

I also updated the docstring a bit, to provide the user with valuable resources about `webdriver_options`.